timemachine/cpp/src/neighborlist.cu

```diff
@@ -218,10 +218,7 @@ void Neighborlist<RealType>::compute_block_bounds_device(
     const int tpb = 32;
     const int B = (N+tpb-1)/tpb; // total number of blocks we need to process
 
-    gpuErrchk(cudaMemsetAsync(d_block_bounds_ctr_, 0, B*D*sizeof(*d_block_bounds_ctr_), stream));
```

Comment:
nice! I didn't realize this was no longer necessary. The kernel directly initializes everything!